### DISCUSSION TITLE
Feature and Enhancement: Add hoverable metrics and notepad for each container (app)

### DISCUSSION THREAD

**Discussion by AuthorShin** [IceWhaleTech/CasaOS#189] [Ideas]
Hello
I always felt the lack of separate metrics for each container (app) because sometimes things are not working and you need data about 2 minutes ago for BookStack, I know it's doable with external apps but I believe that casaOS got the potential for it.
Also having a little notepad for each app (Each app got it's own unique notepad which is containing notes for that app and are different from other ones, so what ever you write in the notepad of 'BookStack' is gonna stay there and only shows when you open that and whatever you write on the notepad of 'NextCloud' is gonna stay and only show up there as well) to remind yourself about what you were/are supposed to do is very nice and handy which can use either Markdown or WYSIWYG editor.
And here is my idea and how you guys might/can implement it in visual form:
[Link to images in high (original) quality](https://xfghxhgh.prodibi.com/a/0kyq268z6qw9rm4?s=sRuJjVV0NOjqS1pBDDCr8ZBmJ5lvEKsFmHQ5rLV2Ziw%3D)
Hover:
By hovering the mouse on the icons on the upper left and bottom it will show a preview inside the app tile:
<img src="https://i.ibb.co/yyNmTKL/final1.png" alt="Flow chart Hover process">

Click:
<img src="https://i.ibb.co/YDVTFRT/Final3.png" alt="Flow chart Click process">

When notepad is open:
<img src="https://i.ibb.co/sPbV2pr/44.png" alt="notepad">

When container (app) metrics is open:
<img src="https://i.ibb.co/9c9hmY3/77.png" alt="container (app) metrics">

**Comment by LinkLeong** [comment]
Is the notepad function intended for use by other members of the family?

**Reply by AuthorShin** [reply]
Yeah, that can be another use case for it.
